config/gocdb_schema.xml

```diff
@@ -583,7 +583,7 @@
         <field>
             <fname>TYPE</fname>
             <length>255</length>
-            <regex>/^X509$/</regex>
+            <regex>/^(X509|OIDC Subject)$/</regex>
```

Comment:
As a note to Future Greg, at the moment when someone/thing accesses the API via OIDC, we just see a OIDC subject - i.e. we don't see the issuer (we do see the issuer when someone uses the portal.) So I think it makes sense to refer to "OIDC Subject" with regards to API credentials.

I wonder if changing `X509` to `X.509` would break everything for existing credentials...

Reply:
> I wonder if changing X509 to X.509 would break everything for existing credentials...

It does, see: https://github.com/gregcorbett/gocdb/tree/X.509

Reply:
I'm not saying there is a problem here related to APIAuthentication, but a note to check, and anyway name consistency might be a good thing

https://github.com/GOCDB/gocdb/blob/58c2c87b606dc2c58bcc022de9c40e807130572a/config/gocdb_schema.xml#L592

Reply:
Captured in https://github.com/GOCDB/gocdb/pull/324